class.jetpack.php

```diff
@@ -2872,6 +2877,26 @@ public static function jetpack_comment_notice() {
 		);
 	}
 
+	/**
+	 * Show the survey link when the user has just disconnected Jetpack.
+	 */
+	function disconnect_survey_notice() {
+		?>
+		<div class="wrap">
+			<div id="message" class="jetpack-message stay-visible">
+				<div class="squeezer">
+					<h4>
+						<?php echo sprintf(
+							__( 'You have successfully disconnected Jetpack.<br />Would you tell us why? Just <a href="%s">answering two simple questions</a> would help us improve Jetpack.', 'jetpack' ),
```

Comment:
I feel like this should be split into two strings (on either side of the `<br />`), as we may well have reason to re-use the string "You have successfully disconnected Jetpack.", and it will allow us to perform later styling changes without modifying translations